OpenTap.Metrics/MetricManager.cs

```diff
@@ -174,6 +185,10 @@ public static IEnumerable<IMetric> PollMetrics(IEnumerable<MetricInfo> interestS
                 case string v:
                     yield return new StringMetric(metric, v);
                     break;
+                case null when metric.Type.HasFlag(MetricType.Nullable & MetricType.String):
```

Comment:
This does not really make sense since `MetricType.Nullable & MetricType.String` evaluates to 0? 

Should probably be something like `case null when metric.Type.HasFlag(MetricType.Nullable) or metric.Type.HasFlag(MetricType.String):`, but I don't know if this is valid syntax